blocks/library/image/block.js

```diff
@@ -109,7 +109,7 @@ class ImageBlock extends Component {
 	}
 
 	render() {
-		const { attributes, setAttributes, focus, setFocus, className, settings } = this.props;
+		const { attributes, setAttributes, focus, setFocus, className, settings, selectionDisable, toggleSelection } = this.props;
```

Comment:
Minor: I think we should rename this prop to either `disableSelection` or `selectionDisabled` to read better.